cli/config/config.go

```diff
@@ -0,0 +1,74 @@
+package config
+
+import (
+	"io/ioutil"
+	"os"
+
+	"gopkg.in/alecthomas/kingpin.v2"
+	"gopkg.in/yaml.v2"
+)
+
+type getFlagger interface {
+	GetFlag(name string) *kingpin.FlagClause
+}
+
+type Resolver struct {
+	flags map[string]string
+}
+
+func NewResolver(files []string, legacyFlags map[string]string) (*Resolver, error) {
+	flags := map[string]string{}
+	for _, f := range files {
+		b, err := ioutil.ReadFile(f)
+		if err != nil {
+			if os.IsNotExist(err) {
+				continue
+			}
+			return nil, err
+		}
+
+		var m map[string]string
+		err = yaml.Unmarshal(b, &m)
+		if err != nil {
+			return nil, err
+		}
+		for k, v := range m {
+			if new, ok := legacyFlags[k]; ok {
```

Comment:
it appears to be compiling here, but `new` is a keyword in golang and should be renamed

Reply:
Thanks for the catch-up!